.babelrc

```diff
@@ -2,7 +2,9 @@
   "presets": ["@babel/preset-env"],
   "plugins": [
     "@babel/plugin-proposal-object-rest-spread",
+    "@babel/plugin-proposal-optional-chaining",
```

Comment:
This allows us to skip `x && x.y` type checks and replace with `x?.y` which returns undefined if x is falsy instead of throwing an error.